Core/src/org/sleuthkit/autopsy/discovery/search/DomainSearchCacheLoader.java

```diff
@@ -57,36 +57,35 @@
 
     @Override
     public Map<GroupKey, List<Result>> load(SearchKey key) throws DiscoveryException, SQLException, TskCoreException, InterruptedException {
-
         List<Result> domainResults = getResultDomainsFromDatabase(key);
-
         // Apply secondary in memory filters
         for (AbstractFilter filter : key.getFilters()) {
             if(Thread.currentThread().isInterrupted()) {
                 throw new InterruptedException();
-            }
-            
+            }      
             if (filter.useAlternateFilter()) {
                 domainResults = filter.applyAlternateFilter(domainResults, key.getSleuthkitCase(), key.getCentralRepository());
             }
         }
-        
         // Grouping by CR Frequency, for example, will require further processing
         // in order to make the correct decision. The attribute types that require
         // more information implement their logic by overriding `addAttributeToResults`.
         List<AttributeType> searchAttributes = new ArrayList<>();
         searchAttributes.add(key.getGroupAttributeType());
         searchAttributes.addAll(key.getFileSortingMethod().getRequiredAttributes());
-        
         for (AttributeType attr : searchAttributes) {
             if(Thread.currentThread().isInterrupted()) {
                 throw new InterruptedException();
-            }
-            
+            }       
             attr.addAttributeToResults(domainResults, 
                     key.getSleuthkitCase(), key.getCentralRepository());
         }
-
+        // Apply secondary in memory filters
+        for (AbstractFilter filter : key.getFilters()) {
+            if (filter.useAlternateFilter()) {
```

Comment:
There was a thread interrupt check here that got removed with the merge conflict.

Reply:
woops, good catch. Thank you